featurevis/image_reader.py

```diff
@@ -2,20 +2,21 @@
 A utility file that takes all numpy arrays in the input folder and saves
 them as pngs in the output folder
 """
+
 from pathlib import Path
 import numpy as np
 from tensorflow import keras
 
 
-def save_npy_as_png(path):
+def save_npy_as_png(input_path, output_path = "lune/outputs"):
```

Comment:
In general, I think this change is good. But I don't think we should have an outputs folder inside the project. If this is to be used as a library, this does not really make sense. So I would say this should probably not have a default value.